auth_ldaps/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2017 Creu Blanca
+# Copyright (C) 2018 Brainbean Apps
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'LDAPS authentication',
+    'version': '12.0.1.0.0',
+    'category': 'Tools',
+    'website': 'https://github.com/OCA/server-auth',
+    'author':
+        'Braibean Apps (https://brainbeanapps.com), '
+        'Creu Blanca, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'application': False,
+    'summary': 'Allows to use LDAP over SSL authentication',
+    'depends': [
```

Comment:
The external dependency on ldap is missing
```python
'external_dependencies': {
        'python': [
            'ldap',
        ],
    },
```

And put it in a separate commit (no squash) to backport it easily on 11.0. :smirk:

Reply:
It should not be assumed that dependency is resolved by transitivity. The best practice is to declare the dependency everywhere it's used.  In your module you explicitly import the ldap module. Therefore you have a direct dependency on this module and it should be declared into the manifest.

Reply:
Done

Reply:
Thank you @alexey-pelykh